apps/admin/frontend/src/screens/write_ins_transcription_screen.tsx

```diff
@@ -305,6 +306,23 @@ export function WriteInsTranscriptionScreen({
   });
   const imageData = imageDataQuery.data ? imageDataQuery.data[0] : undefined;
 
+  // prefetch the next write-in image
+  const queryClient = useQueryClient();
+  const apiClient = useApiClient();
+  useEffect(() => {
+    const nextAdjudication = adjudications[offset + 1];
+    if (nextAdjudication) {
+      void queryClient.prefetchQuery({
```

Comment:
Sanity check: What happens if the prefetch fails? Does react-query's default error handling take care of this (even with the void)?

Reply:
I just tried, experimentally, by returning a rejected promise from the `queryFn` just as `grout` would. I found that if the prefetch fails then I see the error in the console but it doesn't crash anything. I'm not sure what that means w.r.t. React Query's default error handling. Stack trace:
```
  | onError | @ | query.ts:438
  | reject | @ | retryer.ts:112
  | (anonymous) | @ | retryer.ts:172
  | Promise.catch (async) |   |  
  | run | @ | retryer.ts:152
  | createRetryer | @ | retryer.ts:202
  | fetch | @ | query.ts:450
  | fetchQuery | @ | queryClient.ts:453
  | prefetchQuery | @ | queryClient.ts:496
  | (anonymous) | @ | write_ins_transcription_screen.tsx:317
  | invokePassiveEffectCreate | @ | react-dom.development.js:23487
  | callCallback2 | @ | react-dom.development.js:3945
  | invokeGuardedCallbackDev | @ | react-dom.development.js:3994
  | invokeGuardedCallback | @ | react-dom.development.js:4056
  | flushPassiveEffectsImpl | @ | react-dom.development.js:23574
  | unstable_runWithPriority | @ | scheduler.development.js:468
  | runWithPriority$1 | @ | react-dom.development.js:11276
  | flushPassiveEffects | @ | react-dom.development.js:23447
  | (anonymous) | @ | react-dom.development.js:23324
  | workLoop | @ | scheduler.development.js:417
  | flushWork | @ | scheduler.development.js:390
  | performWorkUntilDeadline | @ | scheduler.development.js:157
```

I think this is a good outcome, but not sure.